core/src/main/java/org/frankframework/core/PipeLine.java

```diff
@@ -674,4 +700,15 @@ public void setAdapterToRunBeforeOnEmptyInput(String s) {
 		adapterToRunBeforeOnEmptyInput = s;
 	}
 
+	/**
+	 * The pipeline of this adapter expects to use the following session keys to be set on call. This
+	 * is for adapters that are called as sub-adapters from other adapters. This serves both for documentation,
+	 * so callers can see what session keys to set on call, and for verification that those session keys are present.
+	 *
+	 * @param expectsSessionKeys Session keys to set on call of the adapter, comma-separated.
```

Comment:
```suggestion
	 * @param expectsSessionKeys Session keys to set on call of the pipeline, comma-separated.
```

Of begrijp ik het niet goed?

Reply:
Hij stond voorheen bij de adapter class, ik denk dat dit een copy paste foutje is 😄

Reply:
Dat was inderdaad een copy-paste foutje.